docs/pages/academy/3-schemas/4-schema-building-continued.md

```diff
@@ -110,7 +110,28 @@ define
 "distance-from-coast" sub attribute datatype long;
 ```
 
-Congratulations! You have built your first working GRAKN schema! Of course, this is just a starting point and when you start putting data into your knowledge base you will realise that you need to extend the schema (for example you might want to allow companies to have names), but the one you have built is valid and working and could be loaded into GRAKN as is (you will learn how in the next module of the Academy).
+Congratulations! You have built your first working GRAKN schema! Of course, this is just a starting point and when you start putting data into your knowledge base you will realise that you need to extend the schema (for example you might want to allow companies to have names), but the one you have built is valid and working and could be loaded into GRAKN as is (you will learn how in the next module of the Academy). There is one more topic you need to learn about before going on to the module review. Let's have a look at how we can modify a Grakn schema.
+
+
+## Changing the schema
+One of the best features of Grakn schemas is the fact that they are flexible. At any point in time you can add types, attribute to types roles and so on.
+
+But what about deleting a type? Well, that is a bit more delicate. Imagine you have stored information about a thousand dogs in your knowledge base and suddently you delete the type `dog`. What would happen to all those puppies? You wouldn't want to destroy them, would you. For this reason, in order to remove a type you first have to delete all of its instances.
+
+Once you have done that, you are free to modify your schema as you see fit. In order to undo something that is defined in your schema, you need to use the keyword `undefine`. Do you want to remove the type `dog`?
+
+```graql
+undefine dog sub entity;
```

Comment:
This example fails because `dog` doesn't exist. This is a bug with `undefine`! Either change this example, or mark it `graql-test-ignore` with a `<!-- TODO -->` linking [here](https://work.grakn.ai/entity/19001-undefine-dog-sub-entity-doesnt-work).